.eslintrc.js

```diff
@@ -21,7 +21,11 @@ module.exports = {
       // Only lint the English pages because the Markdown for the other languages is auto-generated from English (via Crowdin).
       // But include all languages when running the ESLint extension in VS Code (in case the user has "eslint.validate": ["mdx"]),
       // otherwise there will be random errors since the non-English files don't match any ruleset.
-      files: [`website/pages/${!isVSCode ? 'en/' : ''}**/*.{md,mdx}`],
+      files: [`website/pages/${isVSCode ? '' : 'en/'}**/*.{md,mdx}`],
+      excludedFiles: [
+        `website/pages/${isVSCode ? '' : 'en/'}developing/graph-ts/*.md`,
+        `website/pages/${isVSCode ? '' : 'en/'}querying/graph-client/*.md`,
+      ],
```

Comment:
Should the `.mdx` files in these folders also be excluded? (Why / why not?)

Reply:
I excluded `.md` because they are fetched from GitHub and we have lint errors from them.
`.mdx` are our docs and we can fix lint errors from there

Reply:
Oh, I saw that `graph-ts` had some `.mdx` files, but I realize now that they are not coming from the `graph-tooling` repo. I guess only `README.md` and `CHANGELOG.md` are coming from the repo.

You do have that rule down below, maybe the `x?` should be removed if we only ever plan on pulling `.md` files?

```ts
filePaths: filePaths.filter((filePath) => /\.mdx?$/.test(filePath)),
```

Reply:
removed in last commit